src/include/nodes/parsenodes.h

```diff
@@ -3401,11 +3401,12 @@ typedef struct InlineCodeBlock
 typedef struct CallStmt
 {
 	NodeTag		type;
-	FuncCall   *funccall;		/* from the parser */
+	/* from the parser */
+	FuncCall   *funccall pg_node_attr(query_jumble_ignore);
 	/* transformed call, with only input args */
-	FuncExpr   *funcexpr pg_node_attr(query_jumble_ignore);
```

Comment:
Conflict report:

  <<<<<<< HEAD
  >-------List>---   *outargs pg_node_attr(query_jumble_ignore);
  >-------/* for INSERT ... EXECUTE */
  >-------Oid >--->-------relation;>------>-------/* INSERT target relation */
  >-------List >--   *attrnos; >-->-------/* columns list by attnum */
  >-------/*
  >------- * for nested CallStmt under INSERT ... EXECUTE
  >------- * for example, in query "INSERT INTO t1 EXEC ('EXEC p1')"
  >------- * the CallStmt of the inner "EXEC p1" will have the following two fields
  >------- * filled in based on the outer "INSERT INTO t1 EXEC"
  >------- */
  >-------void >--   *retdesc; >-->-------/* expected TupleDesc of the result rows */
  >-------void >--   *dest; >----->------->-------/* DestReceiver to send the result rows */
  ======= 
  >-------List>---   *outargs;
  >> 11c34b342bd (Show parameters of CALL as constants in pg_stat_statements)
  } CallStmt;